packages/firebase_admob/lib/firebase_admob.dart

```diff
@@ -0,0 +1,178 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:flutter/services.dart';
+import 'package:meta/meta.dart';
+
+enum MobileAdEvent {
+  loaded,
+  failedToLoad,
+  clicked,
+  impression,
+  opened,
+  leftApplication,
+  closed,
+}
+
+typedef void MobileAdListener(MobileAdEvent event);
+
+class MobileAdTargetingInfo {
```

Comment:
If added some doc to the public API. I'll continue to fill the doc out in the next couple of versions.